src/core/metadata/mod.rs

```diff
@@ -47,6 +49,43 @@ impl WalletMetadata {
         &mut self.2
     }
 
+    /// Return a mutable reference to the inner untyped object.
+    /// This is useful for adding additional parameters to the wallet metadata,
+    /// such as `client_id_schemes_supported`.
+    ///
+    /// NOTE: There is a typed method for adding a `client_id_schemes_supported` parameter.
+    /// See:
+    pub fn untyped_object_mut(&mut self) -> &mut UntypedObject {
+        &mut self.0
+    }
```

Comment:
This is not needed, `DerefMut` is implemented for `WalletMetadata`, targeting `UntypedObject`.